src/print-env.spec.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+/* eslint-env mocha */
+const printEnv = require('.')
+const clone = x => JSON.parse(JSON.stringify(x))
+const snapshot = require('snap-shot-it')
+const execa = require('execa')
+const { join } = require('path')
+
+describe('@bahmutov/print-env', () => {
+  let env
+
+  beforeEach(() => {
+    env = clone(process.env)
+    Object.assign(process.env, {
+      FOOXFF: 'foo',
+      FOOXFE: 'bar',
+      FOOXFA: 'baz',
+      BARXFF: 'blanditiis',
+      BARXFE: 'facere',
+      BARXFA: 'enim'
+    })
+  })
+
+  afterEach(() => {
+    process.env = env
+  })
+
+  it('returns only prefixed env vars', () => {
+    snapshot(printEnv(['FOOX']))
+  })
+
+  it('supports multiple prefixes', () => {
+    snapshot(printEnv(['FOO', 'BAR']))
+  })
+
+  describe('bin print-env command', () => {
+    const bin = join(__dirname, '..', 'bin', 'print-env.js')
+    it('prints sorted variables', async () => {
+      const { stdout } = await execa('node', [bin, 'FOOX'], {
+        env: process.env
```

Comment:
we can use https://github.com/sindresorhus/execa#extendenv option to only pass `env` object and not the entire `env + process.env`

Reply:
Better yet, I just added to `process.env` and didn't pass any options to `execa`; see next commit.